txpipe/twopoint.py

```diff
@@ -564,7 +565,113 @@ def read_metadata(self):
 
         return meta
 
-    
+# Aperture Mass class that inherits from TXTwoPoint
+class TXApertureMass(TXTwoPoint):
+
+    name='TXApertureMass'
+    inputs = [
+        ('calibrated_shear_catalog', ShearCatalog),
+        ('calibrated_lens_catalog', HDFFile),
```

Comment:
I think the lens cat is no longer needed here.

Reply:
If I comment this out then I get an error from `_read_nbin_from_tomography`, which seems to require 'calibrated_lens_catalog' to be present.

Reply:
Ah, sorry, you're right..